### PR TITLE
Campus toggle buttons

### LINKED ISSUE #14: US-1: Campus toggle
As a student, I want to be able to toggle between the SGW and Loyola campus maps so that I can view the campuses.

### ACCEPTANCE CRITERIA:
* There is a button that will toggle between the Loyola Campus and the SGW campus
* When a campus is selected, the map is centered on that campus.
* Must match [Mockup](https://www.figma.com/file/1XeyBVJ2XyC6AHDHYwxvdG/WayFinder-SOEN390?node-id=31%3A3904)

### USER FLOW ACCEPTANCE TEST
* On SGW button press, map is set to SGW campus view
* On Loyola button press, map is set to Loyola campus view

### PR DESCRIPTION
**Pull request by osahtout**
Added two buttons at the bottom of the screen to toggle between the SGW and Loyola campus. references #14

### CODE REVIEW

**Review by bit172 — DISMISSED** [review]
LGTM!

File: components/fetchCampusLocation/FetchCampusLocation.component.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            Loyola
+          </Text>
+        </View>
+      </TouchableHighlight>
+    </View>
+  );
+};
+
+/**
+ * TODO: styling might need to be changed @Ragith
```
Comment (cristian-aldea) [inline components/fetchCampusLocation/FetchCampusLocation.component.jsx]: Should be addressed before this pr is merged

File: components/fetchCampusLocation/FetchCampusLocation.component.jsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  },
+  buttonTextNotSelected: {
+    textAlign: "center",
+    padding: 30,
+    fontSize: 20,
+    color: "#AA2B45"
+  }
+});
+
+// const styles = StyleSheet.create({
```
Comment (cristian-aldea) [inline components/fetchCampusLocation/FetchCampusLocation.component.jsx]: Remove commented code

File: components/fetchCampusLocation/FetchCampusLocation.component.jsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  const [isSGW, setIsSGW] = useState(true);
+
+  /**
+   * Changes the boolean value and the coordinates that will change
+   * the map view
+   */
+  const onSGWPressButton = () => {
+    setIsSGW(true);
+    props.regionChange({
+      latitude: 45.495869,
```
Comment (cristian-aldea) [inline components/fetchCampusLocation/FetchCampusLocation.component.jsx]: These hardcoded values should be stored in a separate const.js file somewhere. Leo was doing a similar thing. Please discuss with him

File: components/map/map.component.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import { StyleSheet, Dimensions } from "react-native";
+
+const Map = props => {
+  return <MapView style={styles.mapStyle} region={props.region} />;
+};
+
+getInitialState = async () => {
+  return {
+    region: {
+      latitude: 45.495869,
```
Comment (cristian-aldea) [inline components/map/map.component.jsx]: Same as before, should be stored in a separate file

File: screens/map-screen/map-screen.component.jsx [hunk 5]
```diff
@@ -1,12 +1,35 @@
...
+
+import Map from "../../components/map/map.component";
+import FetchCampusLocation from "../../components/fetchCampusLocation/FetchCampusLocation.component";
 
 class MapScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      region: {
+        latitude: 45.495869,
```
Comment (cristian-aldea) [inline screens/map-screen/map-screen.component.jsx:12]: Same as above

File: screens/map-screen/map-screen.component.jsx [hunk 6]
```diff
@@ -1,12 +1,35 @@
...
+  onRegionChange = newRegion => {
+    this.setState({ region: newRegion });
+  };
+
   render() {
     return (
       <View style={styles.container}>
-        <MapView style={styles.mapStyle} />
+        <Map region={this.state.region} />
+        <FetchCampusLocation regionChange={this.onRegionChange} />
```
Comment (cristian-aldea) [inline screens/map-screen/map-screen.component.jsx]: Another name might be better. Maybe something like "CampusToggleButton"?

**Review by cristian-aldea — CHANGES_REQUESTED** [review]
Looks good, left a few comments that I would like to be addressed before this is merged

File: components/fetchCampusLocation/FetchCampusLocation.component.jsx (hunk 3 above)
Comment (osahtout) [reply components/fetchCampusLocation/FetchCampusLocation.component.jsx]: 1- Jonathan said something about doing it later
2- I discussed the const file with Leo before and we are wondering what the database is for and if we should store it there

File: components/fetchCampusLocation/FetchCampusLocation.component.jsx (hunk 1 above)
Comment (osahtout) [reply components/fetchCampusLocation/FetchCampusLocation.component.jsx]: not an issue, but maybe he has a better way to do it

File: screens/map-screen/map-screen.component.jsx (hunk 5 above)
Comment (osahtout) [reply screens/map-screen/map-screen.component.jsx:12]: this is where the initial location is set, which we all agreed that it'll be the current location that will be fetched using your pr

File: components/map/map.component.jsx (hunk 4 above)
Comment (osahtout) [reply components/map/map.component.jsx]: hmm I don't think this is needed at all

File: components/fetchCampusLocation/FetchCampusLocation.component.jsx (hunk 2 above)
Comment (osahtout) [reply components/fetchCampusLocation/FetchCampusLocation.component.jsx]: thats my bad

**Review by RGPosadas — CHANGES_REQUESTED** [review]
Implementation:
* Functionality-wise, it's a LGTM!
* Matches mockup nicely
* My only comment is that the toggles seem quite big, maybe making them smaller would give us more screen space in the future?

CI:
* Build failing due to pre-commit. It seems `prettier` doesn't properly run on commit all the time, which is quite annoying.
* I know you have pre-commit installed already (unless you deleted the repo since it initializes under .git/hooks/). If so, re-install it: https://github.com/RGPosadas/WayFinder/wiki/Coding-Rules-and-Standards
* If already installed, please run `pre-commit run -a` locally, and try that.

**Review by RGPosadas — APPROVED** [review]
LGTM

**Review by cristian-aldea — APPROVED** [review]
Looks great, thanks for responding to my suggestions

**Pull request merged**